constants.py

```diff
@@ -1,12 +1,40 @@
 import os
 import numpy as np
 
-# Things anyone wishing to use this repository for their
-# own use will want to change
-MEDIA_DIR = os.path.join(
-    os.path.expanduser('~'),
-    "Dropbox (3Blue1Brown)/3Blue1Brown Team Folder"
-)
+env_MEDIA_DIR = None
+MEDIA_DIR = "#ERROR#"
+
+try:
+    env_MEDIA_DIR = os.getenv("MEDIA_DIR")
+except NameError:
+    try:
+        env_MEDIA_DIR = os.environ['MEDIA_DIR']
+    except KeyError:
+        pass
+
+if not (env_MEDIA_DIR is None):
+    MEDIA_DIR = env_MEDIA_DIR
```

Comment:
Since environ is dictionary you could do this without helper variable
```python
if 'MEDIA_DIR' in os.environ:
     MEDIA_DIR = os.environ['MEDIA_DIR']
```
but I'm unsure about safety.

Reply:
I know that I could just use the `os.environ` dictionary, but I prefer using the helpers methods, and even if `os.environ` does not exists I'd like the code not to crash... (see my previous answer)